components/navbar/StatsDropdown.vue

```diff
@@ -1,72 +1,131 @@
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="offers-stats" :to="`/${urlPrefix}/stats`">
-          {{ $t('navbar.offerStats') }}
-        </nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="series-insight" to="/series-insight">
-          Series</nuxt-link
-        >
-      </b-dropdown-item>
-    </b-dropdown>
-    <b-dropdown v-if="chain === 'rmrk'" data-cy="stats" :triggers="['click']">
-      <template #trigger>
-        <div class="navbar-item" data-cy="stats">
-          {{ $t('stats') }}
-        </div>
+    <div v-if="!isMobile">
+      <b-dropdown
+        v-if="chain === 'bsx' || chain === 'snek'"
+        aria-role="list"
+        data-cy="stats"
+        :triggers="['click']">
+        <template #trigger>
+          <div class="navbar-item" data-cy="stats">
+            {{ $t('stats') }}
+          </div>
+        </template>
+        <b-dropdown-item has-link>
+          <nuxt-link
+            data-cy="global-offers"
+            :to="`${
+              accountId
+                ? `/${urlPrefix}/offers?target=${accountId}`
+                : `/${urlPrefix}/offers`
+            }`">
+            {{ $t('navbar.globalOffers') }}
+          </nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="offers-stats" :to="`/${urlPrefix}/stats`">
+            {{ $t('navbar.offerStats') }}
+          </nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="series-insight" to="/series-insight">
+            Series</nuxt-link
+          >
+        </b-dropdown-item>
+      </b-dropdown>
+      <b-dropdown v-if="chain === 'rmrk'" data-cy="stats" :triggers="['click']">
+        <template #trigger>
+          <div class="navbar-item" data-cy="stats">
+            {{ $t('stats') }}
+          </div>
+        </template>
+        <b-dropdown-item has-link aria-role="menu-item">
+          <nuxt-link data-cy="spotlight" to="/spotlight">
+            {{ $t('spotlight.page') }}
+          </nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="series-insight" to="/series-insight">
+            Series</nuxt-link
+          >
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="sales" to="/sales"> Sales</nuxt-link>
+        </b-dropdown-item>
+        <b-dropdown-item has-link>
+          <nuxt-link data-cy="hot" to="/hot"> Hot</nuxt-link>
+        </b-dropdown-item>
+      </b-dropdown>
+    </div>
+
+    <MobileExpandableSection
+      v-if="isMobile"
+      :no-padding="true"
+      :title="$t('stats')">
+      <template>
+        <template v-if="chain === 'bsx' || chain === 'snek'">
+          <b-navbar-item
+            data-cy="global-offers"
+            :to="`${
+              accountId
+                ? `/${urlPrefix}/offers?target=${accountId}`
+                : `/${urlPrefix}/offers`
```

Comment:
There are still many parts that could be reused. For example, this path string could be written to a getter function, so that we do not need to repeat this stuff.

Reply:
Fixed, as an addition, I changed this component to vue composition